src/assets/config/appconfig.json

```diff
@@ -46,7 +46,6 @@
     ]
   },
   "essConfig": {
-    "MaxEncLimit": "250",
-    "MaxEncSelectionLimit":"100"
+    "MaxEncLimit": "250"
```

Comment:
I think we need to keep 'MaxEncSelectionLimit' in the config

Reply:
resolved